app/views/users/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2 class="text-center">User Lists</h2><br/>
+
+<div class='users container'>
+  <% @user.each do |user| %>
+    <%= render user %>
+  <%end%>
+</div>
```

Comment:
- [ ] OPTIONAL: This page is index page, and it has a link to food of each user registered on the page, even though some of them have no public recipes and the link takes to  empty page. I think it would be better to make public recepies page as an index page, this way all links would probably take to another page with data. 

Currently, index page  | When I click on List all user recipe link
------------- | -------------
<img width="1352" alt="Screenshot 2022-07-29 at 21 40 55" src="https://user-images.githubusercontent.com/77038610/181815099-70c8d880-2165-4fc2-95a7-25dd499c0c7e.png">  | <img width="1345" alt="Screenshot 2022-07-29 at 21 41 19" src="https://user-images.githubusercontent.com/77038610/181815154-a22f3536-b8e8-4fa6-95e6-7e1f537ffd08.png">